verilog/analysis/checkers/undersized_binary_literal_rule_test.cc

```diff
@@ -169,6 +169,46 @@ TEST(UndersizedBinaryLiteralTest, DecimalNumbersNeverCare) {
       kTestCases, "");
 }
 
+// There is no good auto-fix test infrastructure in place, so this is
+// somewhat manual.
+// TODO(hzeller): add test infrastructure in linter_test_utils.h
+
+static std::unique_ptr<UndersizedBinaryLiteralRule> CreateConfiguredRule() {
+  auto result = std::make_unique<UndersizedBinaryLiteralRule>();
+  CHECK(result->Configure("bin:true;hex:true;oct:true").ok());
+  return result;
+}
+
+static std::string UndersizeApplyFix(absl::string_view input) {
+  VerilogAnalyzer analyzer(input, "");
+  CHECK(analyzer.Analyze().ok()) << "Parse issue " << input;
+  using verible::LintRunner;
+  LintRunner<verible::SyntaxTreeLintRule> runner(CreateConfiguredRule());
+  const verible::LintRuleStatus rule_status = runner.Run(analyzer.Data(), "");
+  EXPECT_EQ(rule_status.violations.size(), 1);
+  EXPECT_EQ(rule_status.violations.begin()->autofixes.size(), 1);
```

Comment:
Would ASSERT_EQ be more appropriate, to stop execution immediately?

Reply:
It would, but for whatever reason the gtest.h chokes on that, possibly because the `ASSERT_EQ()` is not used inside a `TEST()` macro. Replacing it with `CHECK_EQ()` now.